staker/staker.go

```diff
@@ -417,11 +362,85 @@ func (s *Staker) Initialize(ctx context.Context) error {
 
 		return s.blockValidator.InitAssumeValid(stakedInfo.AfterState().GlobalState)
 	}
+	return s.setupFastConfirmation(ctx)
```

Comment:
do we want to retry if it failed? e.g. if L1 is down temporarily when we boot the node, we'll never issue fast confirmations

Reply:
If this fails, it'd return an error here which would prevent the node from starting up. Next start it'd retry the queries. Other staker queries work the same way.